ui/src/app/pages/workspace/workspace-library.tsx

```diff
@@ -263,6 +263,7 @@ export const WorkspaceLibrary = fp.flow(withUserProfile())(
       try {
         const workspacesReceived =
           await workspacesApi().getPublishedWorkspaces();
+
```

Comment:
These tests don't run unless the UI changes.  This change is only to trigger that.